bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/LightThingHandler.java

```diff
@@ -78,8 +79,13 @@
     private LightState lightStateCache = new LightState();
     private LightState lastCommand = new LightState();
 
+    private final int ct_max;
+    private final int ct_min;
+
     public LightThingHandler(Thing thing, Gson gson) {
         super(thing, gson);
+        ct_max = NumberUtils.toInt(thing.getProperties().get(PROPERTY_CT_MAX), ZCL_CT_MAX);
```

Comment:
Didn't see this before. We try to remove any dependency to Apache Commons. Please use `Integer.parseInt` or `Integer.valueOf`

Reply:
done